products/log4j.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Log4j
+permalink: /log4j
+layout: post
+category: server-app
+link: https://logging.apache.org/log4j/2.x/security.html
+changelogTemplate: https://logging.apache.org/log4j/2.x/changes-report.html#a{{"__LATEST__"}}
+activeSupportColumn: false
+releaseDateColumn: true
+iconSlug: apache
```

Comment:
i know iconslug doesnt have it but log4j has its own logo , so until iconslug uploads a new one for it we can still use this

Reply:
Yeah, went by same reasoning here.